t/t.go

```diff
@@ -95,6 +95,7 @@ var (
 		"comma separated list of packages that needs to be skipped. "+
 			"Package Check uses string.Contains(). Please check the flag carefully")
 	runCoverage = pflag.Bool("coverage", false, "Set true to calculate test coverage")
+	arch        = pflag.String("arch", "x86", "Machine architecture (required for systest/export test)")
```

Comment:
Add comment on which values it accepts. eg. Valid values: [arm, x86 etc.]

Reply:
done